docs/source/developers/release.rst

```diff
@@ -216,123 +216,365 @@ Be sure to go through on the following checklist:
+
+   We use the twine tool to upload wheels to PyPI:
+
+   .. code-block:: Bash
+
+      # dev/release/post-09-python.sh 10.0.0 0
+      dev/release/post-09-python.sh <version> <rc number>
+
+.. dropdown:: Publish Maven packages
+   :animate: fade-in-slide-down
+   :class-title: sd-fs-5
+   :class-container: sd-shadow-md
+
+   - Logon to the Apache repository: https://repository.apache.org/#stagingRepositories
+   - Select the Arrow staging repository you created for RC: ``orgapachearrow-XXXX``
+   - Click the ``release`` button
+
+.. dropdown:: Update R packages
+   :animate: fade-in-slide-down
+   :class-title: sd-fs-5
+   :class-container: sd-shadow-md
+
+   TODO: Move from https://cwiki.apache.org/confluence/display/ARROW/Release+Management+Guide#ReleaseManagementGuide-UpdatingRpackages
+
+.. dropdown:: Update vcpkg port
+   :animate: fade-in-slide-down
+   :class-title: sd-fs-5
+   :class-container: sd-shadow-md
+
+   Open a pull request to vcpkg:
+
+   .. code-block:: Bash
+
+      ## Fork https://github.com/microsoft/vcpkg on GitHub.
+      ## You need to do this only once.
+      ##
+      ## Prepare your fork of https://github.com/microsoft/vcpkg .
+      ## You need to do this only once.
+      # git clone git@github.com:kou/vcpkg.git ../
+      git clone git@github.com:<YOUR_GITHUB_ID>/vcpkg.git ../
+      cd ../vcpkg
+      ./bootstrap-vcpkg.sh
+      ## Add https://github.com/microsoft/vcpkg.git as "upstream" remote.
+      git remote add upstream https://github.com/microsoft/vcpkg.git
+      cd -
+
+      # dev/release/post-14-vcpkg.sh 10.0.0 ../vcpkg
+      dev/release/post-14-vcpkg.sh X.Y.Z <YOUR_VCPKG_FORK>
+
+   This script pushes a ``arrow-X.Y.Z`` branch to your ``microsoft/vcpkg`` fork. You need to create a pull request from the ``arrow-X.Y.Z`` branch with ``[arrow] Update to X.Y.Z`` title on your Web browser.
+
+.. dropdown:: Update Conan port
+   :animate: fade-in-slide-down
+   :class-title: sd-fs-5
+   :class-container: sd-shadow-md
+
+   TODO
+
+.. dropdown:: Bump versions
+   :animate: fade-in-slide-down
+   :class-title: sd-fs-5
+   :class-container: sd-shadow-md
+
+   .. code-block:: Bash
+
+      # dev/release/post-11-bump-versions.sh 10.0.0 11.0.0
+      dev/release/post-11-bump-versions.sh X.Y.Z NEXT_X.NEXT_Y.NEXT_Z
+
+.. dropdown:: Update tags for Go modules
+   :animate: fade-in-slide-down
+   :class-title: sd-fs-5
+   :class-container: sd-shadow-md
+
+   .. code-block:: Bash
+
+      # dev/release/post-10-go.sh 10.0.0
+      dev/release/post-10-go.sh X.Y.Z
+
+.. dropdown:: Update docs
+   :animate: fade-in-slide-down
+   :class-title: sd-fs-5
+   :class-container: sd-shadow-md
+
+   The API documentations are generated in release process. We just need to upload the generated API documentations:
```

Comment:
```suggestion
   The API documentations are generated in the release process. We just need to upload the generated API documentations:
```

Reply:
And you can also remove "API", since this is for all docs, not just API docs

Reply:
OK. Fixed them.